Simple Incremental/Assets/Scripts/Statics/AttackStatEnemy.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+
+public static class AttackStatEnemy

```

Comment:
This is not an AttackStat. All you've done is added a wrapper around "Random.Range(int1, int2+1)"

I do not think this adds value, and instead it makes it harder for another developer to follow.

Reply:
Consider just calling Random.Range directly in your HelperAttackEnemy and eliminate this class.